java-maven-model/pom.xml

```diff
@@ -73,25 +73,29 @@
     </pluginManagement>
     <plugins>
       <plugin>
-        <artifactId>maven-antrun-plugin</artifactId>
-        <version>1.8</version>
-        <executions>
-          <execution>
-            <id>generate-model</id>
-            <phase>generate-sources</phase>
-            <configuration>
-              <target>
-                <mkdir dir="${targetDir}" />
-                <exec executable="xjc">
-                  <arg line="-extension -b ${bindingFile} -p ${javaPackageName} -d ${targetDir} -encoding UTF-8 ${xsdPath}" />
-                </exec>
-              </target>
-            </configuration>
-            <goals>
-              <goal>run</goal>
-            </goals>
-          </execution>
-        </executions>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>jaxb2-maven-plugin</artifactId>
+          <version>2.2</version>
+          <executions>
+              <execution>
+                  <id>xjc</id>
+                  <goals>
+                      <goal>xjc</goal>
+                  </goals>
+              </execution>
+          </executions>
+          <configuration>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)